src/components/form-editor/form-renderer/form-renderer.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useContext, useEffect, useState } from "react";
+import { OHRIForm } from "@ohri/openmrs-ohri-form-engine-lib";
+import { OHRIFormSchema } from "@ohri/openmrs-ohri-form-engine-lib/src/api/types";
+import { SchemaContext } from "../../../context/context";
+
+const FormRenderer: React.FC = () => {
+  const defaultSchema: OHRIFormSchema = {
+    name: "",
+    pages: [],
+    processor: "EncounterFormProcessor",
+    uuid: "xxx",
+    encounterType: "",
+    referencedForms: [],
+  };
+  const patientUUID = "b280078a-c0ce-443b-9997-3c66c63ec2f8";
```

Comment:
This should be driven through a config, what happens if I ran this app against an OpenMRS instance without a patient with uuid: `"b280078a-c0ce-443b-9997-3c66c63ec2f8"`?